tests/codegen/array_integer_equal_comparison.cpp

```diff
@@ -16,28 +16,9 @@ constexpr auto PROG = "struct Foo { int arr[4]; }"
                       "  }"
                       "}";
 
-TEST(codegen, array_integer_equal_comparison)
-{
-  auto bpftrace = get_mock_bpftrace();
-
-  // Force unroll fallback
-  auto feature = std::make_unique<MockBPFfeature>(true);
-  feature->has_loop(false);
-  bpftrace->feature_ = std::move(feature);
-
-  test(*bpftrace, PROG, NAME);
-}
-
 TEST(codegen, array_integer_equal_comparison_no_unroll)
```

Comment:
nit: can probably rename this test to just `array_integer_equal_comparison` as there is no longer an unroll option.